packages/common/CHANGELOG.md

```diff
@@ -2,8 +2,12 @@
 
 ## 9.3.0
 
+### Fixed
+- Added missing fields to `getBlockChainParameters` response. (rootKeys, level1Keys, level2Keys)
+
 ### Added
 
+- `sendUpdateInstruction` to the gRPC Client.
```

Comment:
[the npm registry](https://www.npmjs.com/package/@concordium/common-sdk) states that 9.3.0 is already released?

Reply:
Yes, I think this is some faulty merging :thinking:
I've fixed it.